ztp/source-crs/SriovVrbClusterConfig.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: sriovvrb.intel.com/v1
+kind: SriovVrbClusterConfig
+metadata:
+  name: config
+  namespace: vran-acceleration-operators 
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  drainSkip: $drainSkip # true if SNO, false by default
+  priority: 1
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  acceleratorSelector:
+    pciAddress: $pciAddress
+  physicalFunction:  
+    pfDriver: "vfio-pci"
+    vfDriver: "vfio-pci"
+    vfAmount: 16
+    bbDevConfig: $bbDevConfig
```

Comment:
Perhaps we can add a newline at the EOF here as well?

Reply:
done